lte/gateway/c/oai/lib/s8_proxy/s8_client_api.cpp

```diff
@@ -228,3 +355,23 @@ void send_s8_create_session_request(
         recv_s8_create_session_response(imsi64, sgw_s11_teid, status, response);
       });
 }
+
+static void convert_proto_msg_to_itti_csr(
+    magma::feg::CreateSessionResponsePgw& response,
+    s8_create_session_response_t** s5_response) {
```

Comment:
Can we just use `s8_create_session_response_t*` argument type instead of  `s8_create_session_response_t**` here?